src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/Service2ServiceEnabledIntegrationTest.java

```diff
@@ -1,12 +1,8 @@
 package uk.gov.hmcts.reform.professionalapi.util;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
-import static com.github.tomakehurst.wiremock.client.WireMock.get;
-import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
```

Comment:
Ok, but next tine remember to disable this wildcard feature in your IDE

Reply:
Removed  and disabled the feature in my IDE